kronos-java/src/main/java/com/lyft/kronos/DefaultParam.kt

```diff
@@ -3,10 +3,11 @@ package com.lyft.kronos
 import java.util.concurrent.TimeUnit
 
 object DefaultParam {
-    val NTP_HOSTS = listOf("2.us.pool.ntp.org", "1.us.pool.ntp.org", "0.us.pool.ntp.org")
+    val NTP_HOSTS = listOf("2.us.pool.ntp.org", "1.us.pool.ntp.org", "0.us.pool.ntp.org", "pool.ntp.org")
```

Comment:
Is it a relevant change? Doesn’t fit the goal of the PR from the description.

Reply:
It's not needed given the large 5 second value I put in. The US addresses are optimized for US customers, primarily for Lyft users. But we do have Canadian users as well, and maybe that should be a separate PR to pic the best default host base on region. I can take it out of this PR.

Reply:
Yes, we’ll probably need to use `N.pool.ntp.org` but better to have it in a separate change.